server/server.go

```diff
@@ -1041,6 +1043,18 @@ func (server *Server) wireFrontendBackend(serverRoute *serverRoute, handler http
 		}
 	}
 
+	if len(serverRoute.replacePathRegex) > 0 {
+		log.Debug(serverRoute.replacePathRegex)
+		if sp := strings.SplitN(serverRoute.replacePathRegex, "$>", 2); len(sp) == 2 {
+			handler = &middlewares.ReplacePathRegex{
```

Comment:
Could you create a function `NewReplacePathRegexHandler` in the `ReplacePathRegex` middleware with this code block.

```go
package middlewares

// ...

func NewReplacePathRegexHandler(regex string, replacement string, handler http.Handler) http.Handler {
	return &ReplacePathRegex{
		Regexp:  regexp.MustCompile(strings.TrimSpace(regex)),
		Repl:    strings.TrimSpace(replacement),
		Handler: handler,
	}
}

// ...
```